<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>win</class>
 <widget class="QMainWindow" name="win">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>798</width>
    <height>529</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>798</width>
    <height>334</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Gestion des personnes infectees par le virus CORONA</string>
  </property>
  <property name="windowOpacity">
   <double>1.000000000000000</double>
  </property>
  <property name="whatsThis">
   <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;img src=&quot;:/icones/backgr.jfif&quot;/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#win &gt; #centralwidget{
	border-image: url(:/t1/corona.webp) 0 0 0 0 stretch stretch;
}
#win &gt; #centralwidget &gt; #widget{
	background-color: transparent;
}</string>
  </property>
  <property name="toolButtonStyle">
   <enum>Qt::ToolButtonIconOnly</enum>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>-10</x>
      <y>0</y>
      <width>1111</width>
      <height>601</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>1111</width>
      <height>601</height>
     </size>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>798</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menumenu">
    <property name="title">
     <string>MENU PERSONNES</string>
    </property>
    <widget class="QMenu" name="menumise_a_jour_des_personnes">
     <property name="title">
      <string>mise a jour des personne</string>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>modifier.png</normaloff>modifier.png</iconset>
     </property>
     <widget class="QMenu" name="actionsupp12">
      <property name="title">
       <string>supprimer personne</string>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>supp.png</normaloff>supp.png</iconset>
      </property>
      <addaction name="actionsupp1"/>
      <addaction name="actionsupp2"/>
      <addaction name="actionsupp3"/>
     </widget>
     <widget class="QMenu" name="menumodifier_personne">
      <property name="title">
       <string>modifier personne</string>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>mise a jour.png</normaloff>mise a jour.png</iconset>
      </property>
      <addaction name="actiontelephone"/>
      <addaction name="actionadresse"/>
     </widget>
     <addaction name="actionAjouter"/>
     <addaction name="actionsupp12"/>
     <addaction name="menumodifier_personne"/>
    </widget>
    <widget class="QMenu" name="actionaffiche">
     <property name="title">
      <string>recherche ,affichage</string>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>recherche.png</normaloff>recherche.png</iconset>
     </property>
     <addaction name="actiondicper"/>
     <addaction name="action2_recherche_par_numero_telephone"/>
     <addaction name="action3_recherche_par_indicatif"/>
     <addaction name="action4_recherche_par_nationalite"/>
     <addaction name="action5_recherche_des_personnes_decedes"/>
     <addaction name="action6_recherche_des_personnes_non_decedes"/>
    </widget>
    <addaction name="menumise_a_jour_des_personnes"/>
    <addaction name="actionaffiche"/>
   </widget>
   <widget class="QMenu" name="menumenu_calcul_et_affichage">
    <property name="title">
     <string>MENU CALCUL ET AFFICHAGE</string>
    </property>
    <addaction name="actionafficher_par_nationalite"/>
    <addaction name="actionpersonne_en_quarantaine"/>
    <addaction name="actionpersonnes_decedes"/>
    <addaction name="actionpersonnes_a_risque"/>
   </widget>
   <widget class="QMenu" name="menuGestion_des_maladie">
    <property name="title">
     <string>MENU GESTION DES MALADIES</string>
    </property>
    <widget class="QMenu" name="menumise_a_jour">
     <property name="title">
      <string>mise a jour</string>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>homemaladie.png</normaloff>homemaladie.png</iconset>
     </property>
     <widget class="QMenu" name="menumodifier_les_donnees_d_une_maladie">
      <property name="title">
       <string>modifier les donnees d'une maladie</string>
      </property>
      <addaction name="actiontfgbhy"/>
      <addaction name="actionmodifier_decede"/>
     </widget>
     <addaction name="actionajoutm_2"/>
     <addaction name="actionsupprimer_une_maladie_2"/>
     <addaction name="menumodifier_les_donnees_d_une_maladie"/>
    </widget>
    <widget class="QMenu" name="menurecherche_affichage">
     <property name="title">
      <string>recherche,affichage</string>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>rechmaladie.png</normaloff>rechmaladie.png</iconset>
     </property>
     <addaction name="actioncontenu_du_dictionnaire_maladies_2"/>
     <addaction name="actionrecherche_par_une_maladie"/>
     <addaction name="actionrecherche_maladie_d_une_personne"/>
     <addaction name="actionrecherche_le_pourcentage_de_chaque_maladie_2"/>
     <addaction name="actionrecherche_maladies_de_chaque_personne"/>
    </widget>
    <addaction name="menumise_a_jour"/>
    <addaction name="menurecherche_affichage"/>
   </widget>
   <addaction name="menumenu"/>
   <addaction name="menuGestion_des_maladie"/>
   <addaction name="menumenu_calcul_et_affichage"/>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionAjouter"/>
   <addaction name="actionsupp1"/>
   <addaction name="actionsupp2"/>
   <addaction name="actionsupp3"/>
  </widget>
  <widget class="QToolBar" name="toolBar_2">
   <property name="windowTitle">
    <string>toolBar_2</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="separator"/>
  </widget>
  <action name="actionAjouter">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>ajouter.png</normaloff>ajouter.png</iconset>
   </property>
   <property name="text">
    <string>ajouter personne</string>
   </property>
  </action>
  <action name="actionsupp1">
   <property name="icon">
    <iconset>
     <normaloff>supppersonne.png</normaloff>supppersonne.png</iconset>
   </property>
   <property name="text">
    <string>suppression personne donne</string>
   </property>
  </action>
  <action name="actionsupp2">
   <property name="icon">
    <iconset>
     <normaloff>suppppnatttionnnnaliteee.png</normaloff>suppppnatttionnnnaliteee.png</iconset>
   </property>
   <property name="text">
    <string>suppression des personnes d'une nationalite donnee</string>
   </property>
  </action>
  <action name="actionsupp3">
   <property name="icon">
    <iconset>
     <normaloff>supptell.png</normaloff>supptell.png</iconset>
   </property>
   <property name="text">
    <string>suppression des personnes d'un indicatif donne (telephone)</string>
   </property>
  </action>
  <action name="actiontelephone">
   <property name="icon">
    <iconset>
     <normaloff>supptell.png</normaloff>supptell.png</iconset>
   </property>
   <property name="text">
    <string>telephone</string>
   </property>
  </action>
  <action name="actionadresse">
   <property name="icon">
    <iconset>
     <normaloff>adresse.png</normaloff>adresse.png</iconset>
   </property>
   <property name="text">
    <string>adresse</string>
   </property>
  </action>
  <action name="actiondicper">
   <property name="icon">
    <iconset>
     <normaloff>engtouttt.png</normaloff>engtouttt.png</iconset>
   </property>
   <property name="text">
    <string>1-contenu du dictionnaire personne</string>
   </property>
  </action>
  <action name="action2_recherche_par_numero_telephone">
   <property name="icon">
    <iconset>
     <normaloff>rechtel.png</normaloff>rechtel.png</iconset>
   </property>
   <property name="text">
    <string>2-recherche par numero telephone</string>
   </property>
  </action>
  <action name="action3_recherche_par_indicatif">
   <property name="icon">
    <iconset>
     <normaloff>rechindi.png</normaloff>rechindi.png</iconset>
   </property>
   <property name="text">
    <string>3-recherche par indicatif</string>
   </property>
  </action>
  <action name="action4_recherche_par_nationalite">
   <property name="icon">
    <iconset>
     <normaloff>rechnat.png</normaloff>rechnat.png</iconset>
   </property>
   <property name="text">
    <string>4-recherche par nationalite</string>
   </property>
  </action>
  <action name="action5_recherche_des_personnes_decedes">
   <property name="icon">
    <iconset>
     <normaloff>decede.png</normaloff>decede.png</iconset>
   </property>
   <property name="text">
    <string>5-recherche des personnes decedes</string>
   </property>
  </action>
  <action name="action6_recherche_des_personnes_non_decedes">
   <property name="icon">
    <iconset>
     <normaloff>non dec.png</normaloff>non dec.png</iconset>
   </property>
   <property name="text">
    <string>6-recherche des personnes non decedes</string>
   </property>
  </action>
  <action name="actionajoutm">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icones/ajoutermaladie.png</normaloff>:/icones/ajoutermaladie.png</iconset>
   </property>
   <property name="text">
    <string>ajouter une nouvelle maladie</string>
   </property>
  </action>
  <action name="actionsupprimer_une_maladie">
   <property name="text">
    <string>supprimer une maladie</string>
   </property>
  </action>
  <action name="actionnombre_d_annee">
   <property name="text">
    <string>nombre d'annee</string>
   </property>
  </action>
  <action name="actionmodifier_deces">
   <property name="text">
    <string>modifier deces</string>
   </property>
  </action>
  <action name="actioncontenu_du_dictionnaire_maladies">
   <property name="text">
    <string>contenu du dictionnaire maladies</string>
   </property>
  </action>
  <action name="actionrecherche_par_maladie">
   <property name="text">
    <string>recherche par maladie</string>
   </property>
  </action>
  <action name="actionrecherche_maladies_d_une_personne">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icones/recherche.png</normaloff>:/icones/recherche.png</iconset>
   </property>
   <property name="text">
    <string>recherche maladies d'une personne</string>
   </property>
  </action>
  <action name="actionrecherche_le_pourcentage_de_chaque_maladie">
   <property name="text">
    <string>recherche le pourcentage de chaque maladie</string>
   </property>
  </action>
  <action name="actionrecherche_maladie_de_chaque_personne">
   <property name="text">
    <string>recherche maladie de chaque personne</string>
   </property>
  </action>
  <action name="actionafficher_par_nationalite">
   <property name="text">
    <string>afficher par nationalite</string>
   </property>
  </action>
  <action name="actionpersonne_en_quarantaine">
   <property name="text">
    <string>personne en quarantaine</string>
   </property>
  </action>
  <action name="actionpersonnes_decedes">
   <property name="text">
    <string>personnes decedes</string>
   </property>
  </action>
  <action name="actionpersonnes_a_risque">
   <property name="text">
    <string>personnes a risque</string>
   </property>
  </action>
  <action name="actionenregistrement_fichier_personnes">
   <property name="text">
    <string>enregistrement fichier personnes</string>
   </property>
  </action>
  <action name="actionrecuperation_fichier_personnes">
   <property name="text">
    <string>recuperation fichier personnes</string>
   </property>
  </action>
  <action name="actionenregistrement_fichier_maladies">
   <property name="text">
    <string>enregistrement fichier maladies</string>
   </property>
  </action>
  <action name="actionrecuperation_fichier_maladies">
   <property name="text">
    <string>recuperation fichier maladies</string>
   </property>
  </action>
  <action name="actionajoutm_2">
   <property name="icon">
    <iconset>
     <normaloff>ajoutermaladie.png</normaloff>ajoutermaladie.png</iconset>
   </property>
   <property name="text">
    <string>ajouter une nouvelle maladie</string>
   </property>
  </action>
  <action name="actionsupprimer_une_maladie_2">
   <property name="icon">
    <iconset>
     <normaloff>supp22.png</normaloff>supp22.png</iconset>
   </property>
   <property name="text">
    <string>supprimer une maladie</string>
   </property>
  </action>
  <action name="actioncontenu_du_dictionnaire_maladies_2">
   <property name="icon">
    <iconset>
     <normaloff>rechmaladie.png</normaloff>rechmaladie.png</iconset>
   </property>
   <property name="text">
    <string>contenu du dictionnaire maladies</string>
   </property>
  </action>
  <action name="actionrecherche_par_une_maladie">
   <property name="text">
    <string>recherche par une maladie</string>
   </property>
  </action>
  <action name="actionrecherche_maladie_d_une_personne">
   <property name="text">
    <string>recherche maladie d'une personne</string>
   </property>
  </action>
  <action name="actionrecherche_le_pourcentage_de_chaque_maladie_2">
   <property name="text">
    <string>recherche le pourcentage de chaque maladie</string>
   </property>
  </action>
  <action name="actionrecherche_maladies_de_chaque_personne">
   <property name="text">
    <string>recherche maladies de chaque personne</string>
   </property>
  </action>
  <action name="actionnombre_d_annees">
   <property name="text">
    <string>nombre d'annees</string>
   </property>
  </action>
  <action name="actiontfgbhy">
   <property name="text">
    <string>nombre d'annee</string>
   </property>
  </action>
  <action name="actionmodifier_decede">
   <property name="text">
    <string>modifier decede</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="resources.qrc"/>
  <include location="ressource3/resources2.qrc"/>
  <include location="test.qrc"/>
  <include location="resourcee.qrc"/>
 </resources>
 <connections/>
 <slots>
  <slot>slot1()</slot>
 </slots>
</ui>
